glossary/MongoDB.md

```diff
@@ -0,0 +1,3 @@
+### MongoDB
+
+MongoDB stores data in flexible, JSON-like documents, meaning fields can vary from document to document and data structure can be changed over time
```

Comment:
I propose that you rephrase this to:

`MongoDB is a NoSQL database model that stores data...`